create docker image (#331)
dotnet/interactive: docker-image/readme.md

# >NET interactive docker image

> This folder conains a docker fiel to generate an image to use latest .NET itneractive tool without the need to isntall anacoda on your machine

Reviewer: Spelling: "contains" ... "
```suggestion
This folder conains a docker fiel to generate an image to use latest .NET itneractive tool without the need to isntall anacoda on your machine
```
```suggestion
This folder contains a dockerfile that generates an image with latest .NET Interactive and Jupyter. This lets you try out .NET Interactive's Jupyter experience without needing to install Jupyter directly.
```

## Build instructions

> using the docker file an new image can be built 

Reviewer: ```suggestion
You can build the Docker image by running the following command in this directory:
```

```powershell
> docker build . --tag dotnet-interactive:1.0 --build-args
```

> The container exposes port 8888 and the port range 1100-1200. A different port range can be specified at build time

Reviewer: It might be helpful to explain what these ports are for.

> The container exposes port 8888 and the port range 1100-1200. A different port range can be specified at build time

Reviewer: ```suggestion
The container exposes port 8888 and the port range 1100-1200. A different port range can be specified at build time.
```

```powershell
> docker build . --tag dotnet-interactive:1.0 --build-args HTTP_PORT_RANGE=1000-1100
```

## Run the container

> To run it execute

Reviewer: ```suggestion
To run it execute the following command:
```

```powershell
> docker run --rm -it -p 8888:8888 -p 1100-1200:1100-1200  --name dotnet-interactive-image dotnet-interactive:1.0
```
Rember to match the port range with the range used when building the image.

> A jupyterlab instance will be published on `http://127.0.0.1:8888/` check the output of the docker run command for hte full url with token value.

Reviewer: ```suggestion
A JupyterLab instance will be published on `http://127.0.0.1:8888/`. Check the output of the `docker run` command for the full URL which includes the Jupyter authentication token.
```
